my.test.js

```diff
@@ -0,0 +1,3 @@
+test('adds 1 + 2 to equal 3', () => {
+  expect(1+2).toBe(3);
+});
```

Comment:
Only related to this PR because it comes under the umbrella of dev tooling, I thought it would be good to have tests ready to go, so this works with `yarn test`